### DISCUSSION TITLE
Hey guys, could you please add some auth icons like google, github, facebook etc.? i'm facing problems in installation with material icons. @emotion.js issues!

### DISCUSSION THREAD

**Discussion by imtheaman** [tailwindlabs/heroicons#649] [Ideas]
Thanks for this wonderful icons library.

**Comment by imtheaman** [comment]
I can do it myself too, but busy in building projects. all right.

**Comment by zefir-git** [comment]
Why burden Heroicons with brands/logos when https://simpleicons.org/ is specifically for this purpose
